web/app.py

```diff
@@ -0,0 +1,101 @@
+from flask import Flask, render_template, request
+# pip install flask-socketio
+# pip install gevent-websocket
+from flask_socketio import SocketIO
+import can
+import cantools
+
+app = Flask(__name__)
+app.config['SECRET_KEY'] = 'secret!'
+socketio = SocketIO(app,cors_allowed_origins="*")
+
+# Set up bus
+bustype = 'socketcan'
+channel = 'vcan0'
+
+bus = can.interface.Bus(bustype=bustype, channel=channel)
+
+# Get data from CAN2
```

Comment:
`can_data/CAN2.dbc` is called a literal constant, and in 99% of cases, we try to avoid that. The reason being, we may want to change this value at some point, but we'd have to dig through a bunch of code to find it. 

Instead, we create a variable at the top of our file before any code where we define these constants. I'm fond of using all uppercase variable names in python to denote constants. So, for example, this would become `DBC_FILE="can_data/CAN2.dbc"`

You've already kinda done it with `bustype` and `channel`, but I'd make those uppercase as well.

Don't forget to include these definitions in the readme! It will be important for someone to know how to configure what DBC file to use, or what can interface to use (vcan0 vs can0 vs can1, for example)